mypy/fastparse.py

```diff
@@ -960,8 +960,11 @@ def visit_Name(self, n: ast3.Name) -> NameExpr:
 
     # List(expr* elts, expr_context ctx)
     @with_line
-    def visit_List(self, n: ast3.List) -> ListExpr:
-        return ListExpr([self.visit(e) for e in n.elts])
+    def visit_List(self, n: ast3.List) -> Union[ListExpr, TupleExpr]:
+        expr_list = [self.visit(e) for e in n.elts]  # type: List[Expression]
+        if isinstance(n.ctx, ast3.Store):
```

Comment:
Could you add a quick comment here and in fastparse2 explaining why?

Reply:
Done